stripped/notebooks/Wczytanie_danych_stripped.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
```

Comment:
Czy to również nie jest zduplikowanie notatników?

Reply:
Nie - to jest zrzut kodu. Duplikacja by oznaczała że dwa miejsca posiadają wspólny kod i mogą się niezależnie modyfikować. Tu wersja stripped ma zawsze kod ze swojego odpowiednika. Jest to bardziej jak cache.